registry/api/errcode/register.go

```diff
@@ -0,0 +1,50 @@
+package errcode
+
+import (
+	"fmt"
+	"net/http"
+	"sync"
+)
+
+var (
+	errorCodeToDescriptors = map[ErrorCode]ErrorDescriptor{}
+	idToDescriptors        = map[string]ErrorDescriptor{}
+	groupToDescriptors     = map[string][]ErrorDescriptor{}
+)
+
+// ErrorCodeUnknown is a generic error that can be used as a last
+// resort if there is no situation-specific error message that can be used
+var ErrorCodeUnknown = Register("registry.api.errcode", ErrorDescriptor{
```

Comment:
This would be the global, default "unknown" error. Probably just group "default" or "errcode".